### ISSUE TITLE
Missing Export in types.d.ts

### ISSUE THREAD

**Issue by TerryYoung518** [Hawkbat/VTubeStudioJS#15]
I encountered an error while using IVTSParameter.name, and so on.

Upon examining the `types.d.ts` file, I noticed that the `BaseParameter` type is set to "any." To resolve this issue, I included the following code snippet in `types.d.ts`, resulting in the successful resolution of the error:

```typescript
export interface BaseParameter {
    name: string;
    value: number;
    min: number;
    max: number;
    defaultValue: number;
}
```

**Comment by Hawkbat** [comment]
Good catch, I hadn't noticed that preventing that type from being exported prevented it from surfacing in its dependent types. I'll fix that.

**Comment by Hawkbat** [comment]
Fixed in v3.3.1.
